Sources/SwiftDocC/Model/Rendering/References/ExternalLocationReference.swift

```diff
@@ -0,0 +1,53 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+/// A specialized ``DownloadReference`` used for ``CallToAction`` directives.
```

Comment:
Maybe we should update this to explain this is specifically for external links? It seems like we'll likely have a use for this outside of `CallToAction` directives at some point – so might be good to mention it's currently only used there but it doesn't seem overly specialized to me.

Reply:
I can change the abstract to generalize it, but leave in the full discussion that references `@CallToAction` specifically, since that's what it's used for right now.